content/pytest_maintaining_state.md

```diff
@@ -0,0 +1,178 @@
+---
+Title: Maintaining states while using Pytest
+Date: 2018-09-10 9:30
+Tags: python, pytest
+Category: unit testing
+Slug: test_states
+Author: Michael Bukachi
+Summary: Maintaining states while using pytest
+about_author: Michael is a Software Engineer at Mobidev Kenya Limited
+email: michaelbukachi@gmail.com
+---
+
+## Pre-requisites
+ > - Beginner level Python
+ > - Python2* or Python3* installed
+ > - Basic knowledge on TDD and python UnitTesting
+ > - Pytest installed
+ 
+ 
+ #Intro
```

Comment:
Nuke the space at col 1 (for this line L20 to L48) to fix formatting for the headings and codeblocks in between

Reply:
Okay. I'll make the change and do another pull request